src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/PrintableDocumentService.java

```diff
@@ -15,10 +15,10 @@
 @Service
 public class PrintableDocumentService {
 
-    private final DocumentManagementService documentManagementService;
+    private final UnsecuredDocumentManagementService documentManagementService;
```

Comment:
Inject with interface reference `DocumentManagementService` here instead of implementation class, as only one of the implementation is going to be available based on the config flag.